src/sst/elements/vanadis/vanadis.cc

```diff
@@ -436,16 +435,19 @@ VANADIS_COMPONENT::setHalt(uint32_t thr, int64_t halt_code)
     output->verbose(
         CALL_INFO, 2, 0, "-> Receive halt request on thread %" PRIu32 " / code: %" PRId64 "\n", thr, halt_code);
 
-    if ( thr >= hw_threads ) {
-        // Incorrect thread, ignore? error?
-    }
-    else {
+    // if ( thr >= hw_threads ) {
```

Comment:
Is this fixed? Can we remove this block?

Reply:
This was removed when we were creating new threads to act as warps. And then we changed to using existing threads as warps after which this change was leftover. I will revert to the original.